templates/fargate-service.cfn.yml

```diff
@@ -57,11 +57,20 @@ Parameters:
 
   ServiceUrlPath:
     Type: String
-    Description: The URL path for the service (e.g., /test)
-    Default: /test
-    MinLength: 1
-    MaxLength: 255
-    ConstraintDescription: Value must be between 1 and 255 characters
```

Comment:
This moved to an an optional field because we now support hostname and path based routing. I will add a regex constraint that supports a path or empty.